tile-generator/layers/obs_events/layer.sql

```diff
@@ -9,8 +9,10 @@ RETURNS TABLE(event_id bigint, geometry geometry, distance_overtaker float, dist
       (case when direction_reversed then -1 else 1 end)::int as direction,
       course,
       speed,
-      way_id::bigint as way_id
+      CASE WHEN road.zone IS NULL THEN 'urban' else road.zone END as zone,
+      overtaking_event.way_id::bigint as way_id
     FROM overtaking_event
+    FULL OUTER JOIN road ON (road.way_id = overtaking_event.way_id)
     WHERE ST_Transform(overtaking_event.geometry, 3857) && bbox;
 
```

Comment:
@opatut this way it works without the duplicate column - thank you for the suggestion, that made things way nicer.